### PR TITLE
Added in missing behavior of Scrawled Writing to move around after Go…

### PR DESCRIPTION
**Pull request by jmf9808**
Scrawled writing will start its 15 minute repop timer after the wolfman despawns where it will then move to 1 of 3 locations. I don't know what the actual position is supposed to be for positions 2 and 3 so I used a map on ffxiclopedia and YouTube footage to place them closely to their original spots.

### CODE REVIEW

**Comment by jmf9808** [comment]
Made fixes.

**Comment by TeoTwawki** [comment]
just FYI..you don't have to close and reopen, just pushing automatically updates the pull request.

**Comment by jmf9808** [comment]
Sorry about that, will remember for the future.

**Comment by wrenffxi** [comment]
There's no need to make a zone globals to handle moving the scrawled writings, because they don't move every X minutes.  They just randomize location and hide for 15 minutes whenever the Wolfman dies.  So have the NPC hide itself when it spawns the Wolfman, and add an onDespawn listener to the Wolfman that randomizes the NPC's location.

File: scripts/zones/Oldton_Movalpolos/npcs/Scrawled_Writing.lua [hunk 1]
```diff
@@ -6,19 +6,20 @@
...
-    if ( npcUtil.tradeHas(trade, 4541) and not mob:isSpawned() ) then
-        player:confirmTrade();
-        local x = npc:getXPos();
-        local y = npc:getYPos();
-        local z = npc:getZPos();
-        mob:setSpawn(x-1,y,z)
-        SpawnMob(ID.mob.GOBLIN_WOLFMAN):updateClaim(player);
-        npc:setStatus(dsp.status.DISAPPEAR);
+function onTrade(player, npc, trade)
+    if npcUtil.tradeHas(trade, 4541) and npcUtil.popFromQM(player, npc, ID.mob.GOBLIN_WOLFMAN, {radius = 2, hide = 10}) then
```
Comment (wrenffxi) [inline scripts/zones/Oldton_Movalpolos/npcs/Scrawled_Writing.lua]: i assume this hide = 10 was in there for testing

Comment (jmf9808) [reply scripts/zones/Oldton_Movalpolos/npcs/Scrawled_Writing.lua]: Yeah, changed it back to 900.

**Pull request merged**